packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -660,6 +641,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     private readonly chunkMap: Map<string, string[]>;
 
     private readonly datastoreContexts: DataStoreContexts;
```

Comment:
Shouldn't dataStoreContexts be moved completely into DataStores and any operations on it should go via the DataStores?
Looks like they are both pointing to the one is ContainerRuntime so they are in sync currently.

Reply:
still staging the changes. eventually we'll be able to fully hide the contexts, but not until we move summarization into the datastores